app/Commands/Servers/RebootCommand.php

```diff
@@ -16,63 +16,18 @@ class RebootCommand extends BaseCommand
 
     public function action(): int
     {
-        if ((bool) $this->option('all')) {
-            $this->rebootAll();
-            return self::SUCCESS;
-        }
-
-        $serverId = $this->argument('server_id');
-
-        if (empty($serverId)) {
-            $serverId = $this->askToSelectServer('Which server would you like to reboot');
-        }
-
-        $server = $this->spinupwp->getServer((int) $serverId);
+        $servers = collect();
 
-        if ((bool) $this->option('force') || $this->confirm("Are you sure you want to reboot \"{$server->name}\"?", true)) {
-            $this->rebootServers([$server]);
-        }
-
-        return self::SUCCESS;
-    }
-
-    protected function rebootAll(): void
-    {
-        if ((bool) $this->option('force') || $this->confirm('Are you sure you want to reboot all servers?', true)) {
-            $this->rebootServers($this->spinupwp->listServers()->toArray());
-        }
-    }
-
-    protected function rebootServers(array $servers): void
-    {
-        if (empty($servers)) {
-            return;
-        }
-
-        $events = [];
-
-        foreach ($servers as $server) {
-            try {
-                $eventId  = $server->reboot();
-                $events[] = ["{$eventId}", $server->name];
-            } catch (\Exception $e) {
-                if (count($servers) === 1) {
-                    $this->error("{$server->name} could not be rebooted.");
-                    return;
-                }
+        if ((bool) $this->option('all')) {
```

Comment:
As long is this is getting cleaned up, the cast to `bool` is redundant. Since it does not accept a value, Laravel will always return it as a bool.